static/docs/command-reference/pipeline/show.md

```diff
@@ -52,6 +53,33 @@ instead of stages.
 
 - `-v`, `--verbose` - displays detailed tracing information.
 
+## Paging the output
+
+This command's output is automatically piped to
+[`less`](<https://en.wikipedia.org/wiki/Less_(Unix)>), if available in the
+terminal. (The exact command used is `less --chop-long-lines --clear-screen`.)
+If less is not available (e.g. on Windows), the output is simply printed out.
+
+> It's also possible to
+> [enable Less paging on Windows](/doc/user-guide/running-dvc-on-windows#enabling-paging-with-less).
```

Comment:
Yeah this is the one I meant. All good here but there may be some small issues in other links...